pkg/storage/minio/cataloger.go

```diff
@@ -0,0 +1,86 @@
+package minio
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/gomods/athens/pkg/errors"
+	"github.com/gomods/athens/pkg/observ"
+	"github.com/gomods/athens/pkg/paths"
+	"github.com/minio/minio-go"
+)
+
+// Catalog implements the (./pkg/storage).Cataloger interface
+// It returns a list of modules and versions contained in the storage
+func (s *storageImpl) Catalog(ctx context.Context, token string, pageSize int) ([]paths.AllPathParams, string, error) {
+	const op errors.Op = "minio.Catalog"
+	ctx, span := observ.StartSpan(ctx, op.String())
+	defer span.End()
+	queryToken := token
+	res := make([]paths.AllPathParams, 0)
+	doneCh := make(chan struct{})
+	defer close(doneCh)
```

Comment:
@Zyqsempai I think this is unused. Did I miss it?

Reply:
Oh, you'r right;)